db/sql/migrations/updates/20190111111557_add_protocolVersion_column_to_peers.sql

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+
+/*
+  DESCRIPTION: Add protocolVersion Column to Peers.
+
+  PARAMETERS: None
+*/
+
+ALTER TABLE "peers" ADD COLUMN IF NOT EXISTS "protocolVersion" VARCHAR(20);
```

Comment:
Do we need default value?

Reply:
Outdated and compatible nodes won't have protocolVersion so it will be empty for them. Updated and compatible nodes will have their own, so I think no need for default value. What do you think?
Also @MaciejBaj